File: Component_Reference/src/main/docbook/en-US/extras/exam-Component_Reference-richtree-Basic_usage.js
private TreeNodeImpl<String> stationRoot = new TreeNodeImpl<String>();
private TreeNodeImpl<String> stationNodes = new TreeNodeImpl<String>(); 
private String[] kickRadioFeed = { "Hall & Oates - Kiss On My List",
                                   "David Bowie - Let's Dance",
                                   "Lyn Collins - Think (About It)",
                                   "Kim Carnes - Bette Davis Eyes",
                                   "KC & the Sunshine Band - Give It Up" };

stationRoot.setData("KickRadio");
stationNodes.addChild(0, stationRoot);
for (int i = 0; i < kickRadioFeed.length; i++){
   TreeNodeImpl<String> child = new TreeNodeImpl<String>();
   child.setData(kickRadioFeed[i]);
   stationRoot.addChild(i, child);
}

File: Component_Reference/src/main/docbook/en-US/extras/exam-Component_Reference-richtree-Basic_usage.java
private DataHolderTreeNodeImpl stationRoot;
private DataHolderTreeNodeImpl rootNodes;

public DataHolderTreeNodeImpl getRootNodes() {
    if (rootNodes == null) {
        String[] kickRadioFeed = {"Hall & Oates - Kiss On My List",
                "David Bowie - Let's Dance",
                "Lyn Collins - Think (About It)",
                "Kim Carnes - Bette Davis Eyes",
                "KC & the Sunshine Band - Give It Up"};
        stationRoot = new DataHolderTreeNodeImpl(false, "KickRadio");
        for (int i = 0; i<kickRadioFeed.length; i++) {
            DataHolderTreeNodeImpl child = new DataHolderTreeNodeImpl(true, kickRadioFeed[i]);
            stationRoot.addChild(i, child);
        }
        rootNodes = new DataHolderTreeNodeImpl();
        rootNodes.addChild(0, stationRoot);
    }
    return rootNodes;
}
